atd-vze/src/queries/crashes_changes.js

```diff
@@ -264,20 +256,10 @@ export const RECORD_DELETE_CHANGE_RECORDS = `
       delete_atd_txdot_changes (
           where: {
             record_id: { _eq: $crashId }
-            record_type: { _in: ["crash", "unit", "primaryperson", "person"] }
+            record_type: { _in: ["crash", "unit", "primaryperson", "person", "charges"] }
```

Comment:
this is why the changes table is full of old stale `record_type = charges` records, bc they were never being deleted.

Reply:
can we delete all record_type = charges records?

Reply:
I mean from the existing table, to fix what was being overlooked

Reply:
imo they can go. if we backfill the charges before the new data model to get them back in the crash page UI, all of the changes will be irrelevant anyways.